src/librustc_driver/lib.rs

```diff
@@ -652,12 +657,10 @@ impl RustcDefaultCalls {
                     }
                 }
                 PrintRequest::TargetCPUs => {
-                    let tm = create_target_machine(sess);
-                    unsafe { llvm::LLVMRustPrintTargetCPUs(tm); }
+                    rustc_trans::print_target_cpus(sess);
```

Comment:
Yes, I just didn't realize how when I wrote this code. One advantage of leaving a PR to rot for a week: You get a fresh perspective :smile: